### ISSUE TITLE
Windows server problem

### ISSUE THREAD

**Issue by redgar72** [lenosisnickerboa/csgosl#102]
I went through the initial installation process. Afterwards I hit the restart button as instructed but it never restarted it just went away. After waiting a few minutes I tried to launch it again and now I am getting an error message from Windows Script Host

Script:
C:\Users\Admin\AppData\Local\Temp\Temp1_csgosl-windows.zip\csgosl\csgosl.vbs
Line:     1
Char:     1
Error:    The System cannot find the file specified.
Code:    80070002
Source:    (null)

Does anyone have a fix to this?

**Comment by lenosisnickerboa** [comment]
Looks like you're installing without first unzipping the downloaded
csgosl-windows.zip? If so, instead put the downloaded zip file in e.g. c:\,
unzip it and you will get a new directory c:\csgosl-windows, step into this
dir and now start csgosl.vbs (or csgosl.bat)

On Aug 8, 2017 22:10, "redgar72" <notifications@github.com> wrote:

I went through the initial installation process. Afterwards I hit the
restart button as instructed but it never restarted it just went away.
After waiting a few minutes I tried to launch it again and now I am getting
an error message from Windows Script Host

Script:
C:\Users\Admin\AppData\Local\Temp\Temp1_csgosl-windows.zip\csgosl\csgosl.vbs
Line: 1
Char: 1
Error: The System cannot find the file specified.
Code: 80070002
Source: (null)

Does anyone have a fix to this?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/lenosisnickerboa/csgosl/issues/102>, or mute the thread
